transport-msg.tex

```diff
@@ -986,6 +992,31 @@ \subsubsection{Overview}
 \hline
 \end{tabular}
 
+\msgdef{EVENT_CONFIG0}
+
+This message is sent by the virtio-msg device and there is no response.
+
+\begin{tabular}{|l|l|l|l|}
+\hline
+Type & Offset & Size (bytes) & Content \\
+\hline \hline
+Request & 0 & 0 & no extra data \\
+\hline
+\end{tabular}
+
+The \msgref{EVENT_CONFIG0} message is used when the extra information in 
+\msgref{EVENT_CONFIG} is not available. In general \msgref{EVENT_CONFIG} 
+\emph{SHOULD} be prefered to \msgref{EVENT_CONFIG0} but this is not always
+possible. If the driver side bus implementation is manufacturing notification
+based on polling intervals, it should use this message to notifiy the driver
+side transport layer that it should use \msgref{GET_DEVICE_STATUS} and
+\msgref{GET_CONFIG} to query for device side changes.
+
+This message may also be used when a single configuration change on the device
+side cannot be communicated in one \msgref{EVENT_CONFIG} message due to maximum
+message size and does not desire to represent it as multiple configuration
+changes.
```

Comment:
I read this chapter several time  but this seems to me not obvious for user ( at least to myself)
I would have  one  `VIRTIO_MSG_EVENT_CONFIG` with a generation count field
the device must  wait a `GET_CONFIG` before be able to send another `VIRTIO_MSG_EVENT_CONFIG` 
I would introduce a `VIRTIO_MSG_EVENT_STATUS` with same principle.